src/tabs/presets/CliEngine.js

```diff
@@ -158,9 +159,14 @@ export default class CliEngine
 
     writeLineToOutput(text) {
         if (text.startsWith("###ERROR")) {
-            this.writeToOutput(`<span class="error_message">${text}</span><br>`);
-            this._cliErrorsCount++;
+            if (this._lastCommandIsDefaults) {
+                this.writeToOutput(`<span class="warning_message">${text}</span><br>`);
+            } else {
+                this.writeToOutput(`<span class="error_message">${text}</span><br>`);
+                this._cliErrorsCount++;
+            }
         } else {
+            this._lastCommandIsDefaults = text.toLowerCase().includes(CliEngine.s_outputResetting);
```

Comment:
Is plain text used for logging and the actual logic here?

Reply:
I didn't get the question

Reply:
So instead of sending something like this
```
{
  type: 'error',
  message: '###ERROR some error message',
}
```

we have plain string which is used for display and hope that this value is formatted correctly when we do the logic. Like the meaning of the message is not separate from display.

Like, if for example this would be localised at some point, wouldn't `###ERROR` become `###BŁAD` in polish

Reply:
I see yeah, it could be improved. This part is straight copy-paste from CLI tab.